### ISSUE TITLE
Support for creating a test run for the cases the test contains, instead of creating a test run with all the cases within the given suite

### ISSUE THREAD

**Issue by serhatgemici** [bahmutov/cypress-testrail-simple#24]
My current project type in TestRail doesn't allow multiple test suites, so I have only 1 test suite containing manual and automated test cases together. That's why each time I run my automated tests, I'm getting a test run that's half executed, because not all of the test cases I have are automated. 

It would be really awesome if the pluging would search the spec/feature files for testcase IDs and create a test run that contains only these cases, instead of creating a test run containing all the test cases within the given suite ID.

Testrail api allows to create a selective test run with include_all flag and case_ids, what is to do is, search for the Case IDs in spec files and initiate the TestRail api call like below:
{
	"suite_id": 1,
	"name": "This is a new test run",
	**"include_all": false,
	"case_ids": [1, 2, 3, 4, 7, 8]**
}

**Comment by samelawrence** [comment]
I would like this feature, but rather than explicitly listing the test cases I want to test, when the test run is created in TestRail, it would be nice to only open the run against the test cases with their IDs listed in the Cypress descriptions. I thought that using `--spec` would achieve this, but it does not. My test runs are being opened against every test case in my suite, not limited to those I have automated. My create TestRail run block looks like this:

```yaml
- name: Start TestRail Run 🏃🏻‍♂️
        id: testRail
        run: |
          runName="Test Automation"

          runId=$(npx testrail-start-run \
            --spec './cypress/integration/**/*' \
            --name "${runName}")
          echo "TestRail run id ${runId}"
          # save the run ID as the output from this step
          echo "::set-output name=runId::${runId}"
```

**Comment by serhatgemici** [comment]
I was trying to mean the same thing but guess my explanation was not that clear. :)  I edited the OG comment. 👍🏻

**Comment by serhatgemici** [comment]
Fixed on [3.0.0](https://github.com/bahmutov/cypress-testrail-simple/compare/v2.4.0...v3.0.0) (2022-04-21)
